site/index.html

```diff
@@ -79,8 +82,16 @@
             <div class="row">
                 <div class="col-3"></div>
                 <div class="col-6">
-                    <label for="selected-area">Test suite:&nbsp;</label>
-                    <select id="selected-area" name="selected-area"></select>
+                    <label for="test-selected-area">Test suite:&nbsp;</label>
+                    <select id="test-selected-area" name="test-selected-area"></select>
+                </div>
+                <div class="col-3"></div>
+            </div>
+            <div class="row">
+                <div class="col-3"></div>
+                <div class="col-6">
+                    <label for="date-selected-area">Dates:&nbsp;</label>
+                    <select id="date-selected-area" name="date-selected-area"></select>
```

Comment:
`date-selected-area` is confusing. There is no dependency between the date and area axis (area is short for "focus area", which is now exposed to in the UI as "Test suite"). I would simply call this selection `selected-period`